packages/babel-node/test/index.js

```diff
@@ -0,0 +1,39 @@
+import path from "path";
+import { writeFileSync } from "fs";
+import { commonJS } from "$repo-utils";
+import {
+  buildProcessTests,
+  buildParallelTests,
+} from "@babel/helper-transform-fixture-test-runner";
+
+const { require, __dirname } = commonJS(import.meta.url);
+
+const fixtureLoc = path.join(__dirname, "fixtures");
+const binLoc = path.join(__dirname, "../lib/babel-node");
+
+// eslint-disable-next-line jest/no-export
+export const runParallel = buildParallelTests(
```

Comment:
Is this file meant to be run by Jest? Or should Jest only run the files in batch-test-runner?

If it's meant to be run by Jest, which tests does it contain? Can we move the dummy test from inside `buildParallelTests` to the top-level of this file?

If it's not meant to be run by Jest, can you move it to a `helpers/` folder so that Jest ignores it?

Reply:
Yes it's better if jest doesn't run it.
I tried moving to `helpers`, but the layout doesn't look that readable and we need to change it to `__dirname=="test/helpers"`.
So I renamed it to `index.skip.js` to skip running.